packages/cwe_checker/cwe_checker.0.2/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+name: "cwe_checker"
+version: "0.2"
+synopsis: "BAP plugin collection to detect common bug classes"
+description: """
+cwe_checker is a suite of tools to detect common bug classes such as use of dangerous functions and simple integer overflows. These bug classes are formally known as Common Weakness Enumerations (CWEs).
+"""
+maintainer: "CWE_checker Team <nils-edvin.enkelmann@fkie.fraunhofer.de>"
+authors: [ "Thomas Barabosch <thomas.barabosch@fkie.fraunhofer.de>" "Nils-Edvin Enkelmann <nils-edvin.enkelmann@fkie.fraunhofer.de>" ]
+license: "LGPL-3.0"
+homepage: "https://github.com/fkie-cad/cwe_checker"
+bug-reports: "https://github.com/fkie-cad/cwe_checker/issues"
+dev-repo: "git+https://github.com/fkie-cad/cwe_checker"
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "1.6"}
+  "yojson" {>= "1.6.0"}
+  "bap" {>= "1.6"}
+  "alcotest" {>= "0.8.3"}
+  "core_kernel" {>= "v0.11" & < "v0.12"}
+  "ppx_jane" {>= "v0.11" & < "v0.12"}
+]
+depexts: [
+  "binutils"
+]
+conflicts: [
+  "fkie-cad-cwe-checker" {!= "0.2"}
+]
+build: [
+  [ "dune" "build" "--profile" "release" ]
+]
+install: [
+  [ make "uninstall" ]
+  [ make "clean" ]
+  [ make "all" ]
+]
+remove: [
```

Comment:
The remove step is not needed in opam2. In fact the build uninstall is failing (transformed to a warning by opam) with
```
<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
[WARNING] package uninstall script failed at
          /home/travis/.opam/opam-init/hooks/sandbox.sh remove make uninstall:
          # context     2.0.3 | linux/x86_64 | ocaml-base-compiler.4.05.0 |
          file:///home/travis/build/ocaml/opam-repository
          # path
          ~/.opam/ocaml-base-compiler.4.05.0/.opam-switch/remove/cwe_checker.0.2
          # command     ~/.opam/opam-init/hooks/sandbox.sh remove make
          uninstall
          # exit-code   2
          # env-file    ~/.opam/log/cwe_checker-2123-5ddb38.env
          # output-file ~/.opam/log/cwe_checker-2123-5ddb38.out
          ### output ###
          # dune uninstall
          # The following <package>.install are missing:
          # - _build/default/cwe_checker.install
          # - _build/default/src/cwe_checker_core.install
          # You need to run: dune build @install
          # Makefile:25: recipe for target 'uninstall' failed
          # make: *** [uninstall] Error 1
```

Can you please remove the remove section from the opam file?

Reply:
Good to know! I removed the remove section.